packages/go/ein/ad.go

```diff
@@ -700,12 +720,12 @@ func ParseDomainTrusts(domain Domain) ParsedDomainTrustData {
 				},
 				IngestibleRel{
 					RelProps: map[string]any{
-						ad.IsACL.String():      false,
-						"sidfiltering":         trust.SidFilteringEnabled,
-						"tgtdelegationenabled": trust.TGTDelegationEnabled,
-						"trustattributes":      finalTrustAttributes,
-						"trusttype":            trust.TrustType,
-						"transitive":           trust.IsTransitive},
+						ad.IsACL.String():                false,
+						ad.SidFiltering.String():         trust.SidFilteringEnabled,
+						ad.TGTDelegationEnabled.String(): trust.TGTDelegationEnabled,
+						ad.TrustAttributes.String():      finalTrustAttributes,
+						ad.TrustType.String():            trust.TrustType,
+						"transitive":                     trust.IsTransitive},
```

Comment:
This type doesn't exist as an ad constant

Reply:
Can we add it to the schema?

Reply:
Not a problem, added!